README.md

```diff
@@ -1,10 +1,44 @@
 # xsamtools
-Lightly-modified versions of [htslib](https://github.com/samtools/htslib)
-and [bcftools](https://github.com/samtools/bcftools), available through pypi packaging, with capability to merge VCF
-streams.
+xsamtools makes the [samtools](https://samtools.github.io/) tooling from [htslib](https://github.com/samtools/htslib)
+and [bcftools](https://github.com/samtools/bcftools) available through pypi packaging. These tools have been lightly
+modified to allow merges on VCF streams without an index.
 
-# installation
+# Installation
 ```
 pip install xsamtools
 ```
-A C toolchain is required to install xsamtools.
+Installation requires a C toolchain.
```

Comment:
I would include the dependencies needed for an Ubuntu (any Debian?) system:

```sudo apt install libcurl4-openssl-dev libbz2-dev liblzma-dev```

Reply:
will add